includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -635,11 +636,13 @@ public function delete_resubscribe_meta( $resubscribe_order ) {
 	/**
 	 * Don't transfer Stripe fee/ID meta to renewal orders.
 	 *
-	 * @param int $resubscribe_order The order created for the customer to resubscribe to the old expired/cancelled subscription
+	 * @param WC_Stripe_Order|WC_Order $resubscribe_order The order created for the customer to resubscribe to the old expired/cancelled subscription
 	 */
 	public function delete_renewal_meta( $renewal_order ) {
-		WC_Stripe_Helper::delete_stripe_fee( $renewal_order );
-		WC_Stripe_Helper::delete_stripe_net( $renewal_order );
+		$renewal_order = WC_Stripe_Order::get_by_id( $renewal_order );
+
+		$renewal_order->delete_fee();
+		$renewal_order->delete_net();
```

Comment:
1. I think the docblock is incorrect -- wrong param name description.
2. Isn't the function param `$renewal_order` a `WC_Order` object? In that case, should this be `WC_Stripe_Order::to_instance()` instead of `WC_Stripe_order::get_by_id()`?

Reply:
Oops! You're totally right. I have fixed these issues in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4109/commits/6a0bd6c15a0b824c9113e77587ed720a4f803d1b